sources/packages/web/src/components/students/modals/ConfirmEditApplication.vue

```diff
@@ -21,22 +20,42 @@ import { useModalDialog } from "@/composables";
 import { defineComponent } from "vue";
 
 export default defineComponent({
+  props: {
+    beforeEdit: {
```

Comment:
I would recommend to follow boolean naming convention and close to component's own purpose.

something like `IsConfirmationOnSubmit` or `IsConfirmationOnEdit`

Reply:
updated